docs/src/components/footer.js

```diff
@@ -50,27 +51,27 @@ const UnorderedList = ({ list }) => (
 )
 
 const firstColumn = [
-  { text: 'Support', href: 'https://notify.gov.au/support' },
+  { text: 'Support', href: notifyUrl('support') },
```

Comment:
I think this should be `notifyUrl('/support')` with the leading slash. One so it's obvious that it's a path (readability and findability [e.g. search for `/support` URLs]) and two because it's a path which I think should start with a slash.